devlib/utils/misc.py

```diff
@@ -416,24 +417,50 @@ def convert_new_lines(text):
     """ Convert new lines to a common format.  """
     return text.replace('\r\n', '\n').replace('\r', '\n')
 
+def sanitize_cmd_template(cmd):
+    msg = (
+        '''Quoted placeholder should not be used, as it will result in quoting the text twice. {} should be used instead of '{}' or "{}" in the template: '''
```

Comment:
Is the message missing the command placeholder?

Reply:
It somehow got lost, I'll concatenate the strings instead, to avoid double curly braces in the message string so it can be grepped more easily